cypress/integration/css/accordion.spec.ts

```diff
@@ -0,0 +1,31 @@
+describe('Accordion', () => {
+  it('should have ".open" and "[aria-expanded=true]" for open sections and vice versa', () => {
+    cy.visit('http://localhost:56206/iframe.html?id=accordion');
+
+    cy
+      .get('.dso-accordion-section:nth-child(3)')
+      .should('have.class', 'dso-open')
+      .find('.dso-section-handle > a')
```

Comment:
alleen de `a` variant?

Reply:
We demo-en nergens de `button` variant, dus dan is het lastig testen. Dan heb ik liever dat de test wordt aangepast als we de functionaliteit demo-en